src/routes/transactions/converters/details.rs

```diff
@@ -28,7 +28,10 @@ impl MultisigTransaction {
             tx_status: self.map_status(&safe_info),
             tx_info: self.transaction_info(info_provider).await,
             tx_data: Some(TransactionData {
-                to: AddressEx::address_only(&self.safe_transaction.to),
+                to: info_provider
```

Comment:
Do you think it makes sense to have an helper or flag in the method for this? (default on failure)

Reply:
I thought about the helper method. as in `AddressEx::any_source` but I would need to pass in the `info_provider` anyway, which is not that bad, jsut thought it would be a bit too roundabout. 

Btw, I noticed we already return this info in `txInfo.to` 🤔 it appears we are duplicating this here.

Reply:
The clients have a single screen showing `txData` therefore redundancy makes sense.